myhpi/core/migrations/0013_merge_first_and_second_level_menu_items.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.9 on 2024-04-06 17:18
+from django.contrib.contenttypes.models import ContentType
+from django.db import migrations
+
+
+def replace_menu_items(apps, schema_editor):
+    BasePage = apps.get_model("core", "BasePage")
+    MenuItem = apps.get_model("core", "MenuItem")
+    menu_item_content_type = ContentType.objects.get_for_model(MenuItem)
+    FirstLevelMenuItem = apps.get_model("core", "FirstLevelMenuItem")
+    SecondLevelMenuItem = apps.get_model("core", "SecondLevelMenuItem")
+    for menu_item_data in FirstLevelMenuItem.objects.values():
```

Comment:
this could also be `for menu_item in FirstLevelMenuItem.objects.all():`, making the next line obsolete